content/actions/learn-github-actions/variables.md

```diff
@@ -69,16 +69,16 @@ jobs:
       Greeting: Hello
     steps:
       - name: "Say Hello Mona it's Monday"
-        run: echo "$Greeting $First_Name. Today is $DAY_OF_WEEK!"
+        run: echo "${{ env.Greeting }} ${{ env.First_Name }}. Today is ${{ env.DAY_OF_WEEK }}!"
```

Comment:
I don't think we should change this to use the `env` context because this section is about runner environment variables. However, this is obviously not clear enough, so I'll suggest a change below to make this clearer.

Change this back to:

```suggestion
        run: echo "$Greeting $First_Name. Today is $DAY_OF_WEEK!"
```